### ISSUE TITLE
Distinguish between tslint warnings and errors

### ISSUE THREAD

**Issue by KurtPreston** [ThaNarie/tslint-teamcity-reporter#6]
Currently, tslint-teamcity-reporter reports lint warnings as errors, and fails the build.

It would be nice for it to respect when a tslint rule is given a severity of `warning`, and report it differently to TeamCity

**Comment by KurtPreston** [comment]
The `eslint-teamcity` project supports warnings. Assuming tslint behaves similarly, the code is a good example implementation:
https://github.com/andreogle/eslint-teamcity/blob/master/src/formatters/errors.js

**Comment by ThaNarie** [comment]
Thanks for brining this up, I'll have a look at it!

**Comment by ThaNarie** [comment]
Hi @KurtPreston, I've rebuilt the plugin with the requested feature. I've followed the eslint way of reporting in teamcity (grouped by file instead of each individual error).

Please check the readme for updates/info, and let me know if the `3.x` version works for you!

**Comment by KurtPreston** [comment]
@ThaNarie Cool!  I'm trying it out, but having difficulties.

I'm pointing to the 3.0 repo with this line in my `package.json`:

```json
"tslint-teamcity-reporter": "https://github.com/ThaNarie/tslint-teamcity-reporter.git#814a74",
```

I'm running the linter with:
```javascript
 tslint({
  program,
  formattersDirectory: `${projectRoot}/node_modules/tslint-teamcity-reporter/`,
  formatter: 'tslint-teamcity-reporter'
});
```

However, when I run, it gives the following error:
```
Error: formatter 'tslint-teamcity-reporter' not found
```

Any idea what I'm doing wrong?

**Comment by ThaNarie** [comment]
@KurtPreston it's available on npm, please use that :)
The repo doesn't contain build files, so won't work directly.

**Comment by KurtPreston** [comment]
Ah, didn't see that version listed yesterday.  Okay, running on `3.0.1`, I get the error:

```
[10:53:31] Error: Cannot find module '@babel/runtime/core-js/object/values'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/kpreston/Code/salsa/node_modules/tslint-teamcity-reporter/lib/Reporter.js:10:38)
```

**Comment by KurtPreston** [comment]
FWIW, looks like my @babel/runtime is on `7.1.5`

**Comment by ThaNarie** [comment]
The package still had the beta-version of runtime specified, just updated everything babel related and published `3.1.0`, I hope it all works out this time :)

**Comment by KurtPreston** [comment]
Works great, thanks!
